UI/src/app/home/home.component.ts

```diff
@@ -14,7 +14,7 @@ export class HomeComponent implements AfterViewInit, OnInit {
     @ViewChild("textInput") textInput: TextInputComponent;
     @ViewChild("chart") chart: ChartComponent;
 
-    data: Trace[];
+    data: Trace;
```

Comment:
why did you remove the Array? we could display more than one trace, so we need the array of traces? 🤔

Reply:
Yes, It also need to change back to trace[];